schema/sql/reserved.sql

```diff
@@ -0,0 +1,62 @@
+-- -*- mode: sql; sql-product: postgres -*-
+-- An helper for bulk-importing the data
+create temp table tmp_table (
+    "name" text unique not null
+);
+
+create function pg_temp.import_reserved() returns void
```

Comment:
When/how is the `pg_temp` schema created?

Reply:
no idea. Once again something someone did in prod without going through version control

Reply:
This is how it was, however this is just a function and works as is, populating everything to tmp_table then into the db under reserved_usernames